blockchain/state/statedb.go

```diff
@@ -90,6 +90,8 @@ type StateDB struct {
 	validRevisions []revision
 	nextRevisionId int
 
+	prefetching bool
```

Comment:
If this is declared as public member variable, `OpenTrieForPrefetching` and `OpenStorageTrieForPrefetching
` won't be needed, as it can be handled inside `OpenTrie` and `OpenStorageTrie`.

```suggestion
	Prefetching bool
```

Reply:
Even if making this variable as public, we need to pass the value to the function somehow. In those functions `OpenTrie` and `OpenStorageTrie`, there is no way to access `StateDB.Prefetching` directly.

Reply:
Oh, you're right. I mixed `statedb.Database` up with `state.StateDB` ... 😅